engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-errata.html

```diff
@@ -53,8 +53,8 @@ <h3 translate ng-show="selectedErrataOption === 'current'">Installable Errata</h
 
     <span data-block="list-actions" ng-hide="contentHost.readonly">
       <div bst-modal="applySelected()" model="host">
-        <div data-block="modal-header" translate>Apply Errata to Content Host "{{host.name}}"?</div>
-        <div data-block="modal-body" translate>Are you sure you want to apply Errata to content host "{{ host.name }}"?</div>
+        <div data-block="modal-header" translate>Apply Errata to Content Host "{{host.displayname}}"?</div>
+        <div data-block="modal-body" translate>Are you sure you want to apply Errata to content host "{{ host.displayname }}"?</div>
```

Comment:
@lfu Do you think this should be {{host.display_name}}?

Reply:
You are correct!